containers/ddev-php-base/generic-files/usr/local/bin/build_php_extension.sh

```diff
@@ -36,12 +36,16 @@ else
   PECL_EXTENSION="${EXTENSION_NAME}-${EXTENSION_VERSION}"
 fi
 
+timeout "${START_SCRIPT_TIMEOUT:-30}" pecl channel-update pecl.php.net || true
+
+echo "Building php${PHP_VERSION}-${EXTENSION_NAME}..."
+# TODO: Some extensions ask for configuration options
+# We may want to have some options set, but for now let's accept default settings with yes
+(yes '' | pecl -d php_suffix="${PHP_VERSION}" install -f "${PECL_EXTENSION}" >/dev/null) || true
```

Comment:
This is probably the most important part in building.
Xdebug didn't ask anything.
But extensions like apcu and redis ask if you want to enable/disable support for something.